document-service/documents-ui/src/components/documentRecord/index.vue

```diff
@@ -106,7 +106,7 @@ const isScanPending = computed(() => documentRecord.value.consumerDocumentId.len
           {{ formatToReadableDate(documentRecord.consumerFilingDateTime, true) || 'Not Entered' }}
         </span>
         <span class="font-bold">{{ $t('documentReview.labels.author') }}</span>
-        <span class="col-span-2">{{ documentRecord?.author }}</span>
+        <span class="col-span-2">{{ documentRecord?.author || scanningDetails?.author || 'N/A' }}</span>
 
```

Comment:
Apologies for not replying to your comment earlier. I've made this update in my PR as well. 😊

Reply:
No prob at all sir!